dist/javascript.js

```diff
@@ -91,6 +91,9 @@ return JSON.stringify({
         };
 
         return noscript_data;
-    })()
+    })(),
 
+    workers: (() => {
```

Comment:
Similarly I think we can rely on the `observers.js` custom metric to extract this value. Otherwise LGTM.